pkg/scheduler/util/topologies.go

```diff
@@ -39,15 +39,11 @@ func GetNamespacesFromPodAffinityTerm(pod *v1.Pod, podAffinityTerm *v1.PodAffini
 
 // PodMatchesTermsNamespaceAndSelector returns true if the given <pod>
 // matches the namespace and selector defined by <affinityPod>`s <term>.
-func PodMatchesTermsNamespaceAndSelector(pod *v1.Pod, namespaces sets.String, tenants sets.String, selector labels.Selector) bool {
+func PodMatchesTermsNamespaceAndSelector(pod *v1.Pod, namespaces sets.String, selector labels.Selector) bool {
```

Comment:
Do we have an issue to track the work to add the tenant back?

Reply:
This is a part of PodAffinityTerm. Arktos code in master never had tenants in this struct. It might be a bug or missing feature in Arktos multi-tenancy. This PR only reserve current master behavior. We can discuss later whether PodAffinityTerm needs to have tenant name as a selector.

Reply:
https://github.com/CentaurusInfra/arktos/issues/1100